src/app/api/common/votableSupply/getVotableSupply.ts

```diff
@@ -4,11 +4,18 @@ import { cache } from "react";
 
 async function getVotableSupply() {
   const { namespace } = Tenant.current();
-  const votableSupply = await prisma[`${namespace}VotableSupply`].findFirst({});
-  if (!votableSupply) {
-    throw new Error("No votable supply found");
+  try {
+    const votableSupply = await prisma[`${namespace}VotableSupply`].findFirst(
+      {}
+    );
+    return votableSupply?.votable_supply || "0";
+  } catch (error) {
+    console.log("Error fetching votable supply:", error);
+
+    // Handle prisma errors for new tenants when votable supply is not available
+    // 1 is returned to avoid zero division errors
+    return "1";
```

Comment:
Same here. New tenant edge case. Need to return 1 in order to avoid zero division error.